index.php

```diff
@@ -2,13 +2,41 @@
     /* -*- coding: utf-8; mode: php; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:set fenc=utf-8 filetype=php et sw=4 ts=4 sts=4: */
     /* Copyright (c) 2007, 2014, The MacPorts Project. */
     include_once("includes/common.inc");
-    print_header('The MacPorts Project -- Home', 'utf-8');
+    print_header('The MacPorts Package Manager for macOS', 'utf-8');
```

Comment:
I am a bit conflicted about "Package Manager" as MacPorts is a ports system. The difference might not be significant anymore...?

The term "The MacPorts Project" has always been the official name after the DarwinPorts rename and I guess that was well chosen with legal obligations in mind.

Reply:
I'm using macports for years (thanks!), I don't know the difference between a ports system and a package manager. But this could be explained in an detailed page "what make MacPort unique" or "what is macport". Reading https://en.wikipedia.org/wiki/FreeBSD_Ports "The FreeBSD Ports collection is a package management system for the FreeBSD operating system".

My title proposition could certainly be improved, but it has to contain keyword people search for and the description on google need to feel familiar and easy. Today it's not very good:
![Screenshot 2021-02-07 at 16 54 49](https://user-images.githubusercontent.com/164074/107151925-8c379b80-6965-11eb-9fa9-f35e5514579b.png)

About using "The MacPorts Project", I understand the concern. Is this still a real concern? In day to day life people refer to "macports" right?